lib/datasource/github-releases/index.spec.ts

```diff
@@ -67,4 +73,89 @@ describe(getName(), () => {
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
   });
+
+  describe('getDigest', () => {
+    const lookupName = 'some/dep';
+    const currentValue = 'v1.0.0';
+    const currentDigest = 'v1.0.0-digest';
+
+    const releaseMock = new GitHubReleaseMocker(githubApiHost, lookupName);
+
+    it('requires currentDigest', async () => {
+      const digest = await getDigest({ datasource, lookupName }, currentValue);
+      expect(digest).toBeNull();
+    });
+
+    it('defaults to currentDigest when currentVersion is missing', async () => {
+      const digest = await getDigest(
+        {
+          datasource,
+          lookupName,
+          currentDigest,
+        },
+        currentValue
+      );
+      expect(digest).toEqual(currentDigest);
+    });
+
+    // TODO: reviewers - this is awkward, but I found returning `null` in this case to not produce an update
+    // I'd prefer a PR with the old digest (that I can manually patch) to no PR, so I made this decision.
+    it('ignores failures verifying currentDigest', async () => {
+      releaseMock.release(currentValue);
+      const digest = await getDigest(
+        {
+          datasource,
+          lookupName,
+          currentValue,
+          currentDigest,
+        },
+        currentValue
+      );
+      expect(digest).toEqual(currentDigest);
+    });
+
+    describe('with cache', () => {
+      let tmpDir: DirectoryResult;
+      beforeEach(async () => {
+        tmpDir = await dir();
+        packageCache.init({ cacheDir: tmpDir.path });
+        memCache.init();
```

Comment:
Cache needs to be disabled after each test to not cause side effects if tests are running out of order.

@rarkins i don't think we need to test caching here at all?

Reply:
We can skip cache testing

Reply:
@thepwagner Please remove the cache tests and add some ignore coverage comments if required